04-advanced-topics/quantile_normalize_own_data_adv_topics_01.Rmd

```diff
@@ -175,10 +175,10 @@ exprs_long_df <- expression_df %>%
 
```

Comment:
Looks like `library(ggplot2)` also needs to be at the top of this notebook.

Reply:
Same comment as above.